src/tuplet.ts

```diff
@@ -85,20 +83,14 @@ export class Tuplet extends Element {
 
   protected options: TupletOptions;
   protected numNotes: number;
-  protected point: number;
 
   protected bracketed: boolean;
-  protected yPos: number;
-  protected xPos: number;
-  protected width: number;
-
+  protected txtElement: Element;
```

Comment:
Is this for the number? (not the lines)

Can we rename it `textElement` since it's not saving that many characters anyways?

Reply:
done